packages/@react-aria/live-announcer/src/LiveAnnouncer.tsx

```diff
@@ -20,17 +20,28 @@ let node = null;
 let clearTimeoutId = null;
 const LIVEREGION_TIMEOUT_DELAY = 1000;
 
+type TAriaLive = 'assertive' | 'off' | 'polite' | undefined;
+
+interface IMessageProps {
```

Comment:
we don't use the "I" naming convention
this should be `MessageProps` without the "I" I think

Reply:
got it!